intermediate-algorithm-scripting-yahtzee-game/index.js

```diff
@@ -1,9 +1,19 @@
-"use strict";
-
-// Sort the array elements in ascending order
-// Important test case: input array should not change as arrays are passed by reference
+const scoreInputs = document.querySelectorAll("#score-options input");
+const scoreSpans = document.querySelectorAll("#score-options span");
+const scoreLabels = document.querySelectorAll("#score-options label");
+const diceButtons = document.querySelectorAll("#dice > button");
+const diceContainer = document.getElementById("dice");
+const currentRoundText = document.getElementById("current-round");
+const currentRoundRollsText = document.getElementById("current-round-rolls");
+const totalScoreText = document.getElementById("total-score");
+const scoreHistory = document.getElementById("score-history");
+const rollDiceBtn = document.getElementById("roll-dice-btn");
+const keepScoreBtn = document.getElementById("keep-score-btn");
+
+// Good review of selection Sort algorithm
```

Comment:
Agreed, it would be good to review selection sort here. Also, it's good that it introduces the concept of creating deep copies of incoming arrays.